src/ray/gcs/pubsub/gcs_pub_sub.cc

```diff
@@ -44,17 +44,12 @@ Status GcsPubSub::SubscribeAll(const std::string &channel, const Callback &subsc
   return SubscribeInternal(channel, subscribe, done);
 }
 
-Status GcsPubSub::Unsubscribe(const std::string &channel_name, const std::string &id) {
-  std::string pattern = GenChannelPattern(channel_name, id);
-
-  absl::MutexLock lock(&mutex_);
-  // Add the UNSUBSCRIBE command to the queue.
-  auto channel = channels_.find(pattern);
-  RAY_CHECK(channel != channels_.end());
-  channel->second.command_queue.push_back(Command());
+Status GcsPubSub::Unsubscribe(const std::string &channel, const std::string &id) {
+  return UnsubscribeInternal(channel, id);
+}
 
-  // Process the first command on the queue, if possible.
-  return ExecuteCommandIfPossible(channel->first, channel->second);
+Status GcsPubSub::UnsubscribeAll(const std::string &channel) {
```

Comment:
Why is this change needed? No one seems to call it.

Reply:
I hava remove this change, thanks.